cmake/SetupCharm.cmake

```diff
@@ -28,7 +28,7 @@ endif(NOT TARGET Charmxx)
 #       with GCC
 string(
     REGEX REPLACE "<CMAKE_CXX_COMPILER>"
-    "${CHARM_COMPILER} -pthread -module DistributedLB -no-charmrun"
+    "${CHARM_COMPILER} -pthread -module CommonLBs -no-charmrun"
```

Comment:
Unclear to me if we want to merge this commit now as part of this PR, or later after the registration PR when more pieces are in. Thoughts?

Reply:
I think this was mainly a stub/placeholder so I have no objection to merging now